addons/website_livechat/tests/test_livechat_basic_flow.py

```diff
@@ -130,7 +131,8 @@ def _common_basic_flow(self):
         self.assertEqual(channel.livechat_active, True, "The livechat session must be active as the visitor did not left the conversation yet.")
 
         # Post message from operator
-        self._send_message(channel, self.operator.email, "Message from Operator", author_id=self.operator.partner_id.id)
+        with freeze_time(fields.Datetime.to_string(fields.Datetime.now() + datetime.timedelta(hours=1))):
```

Comment:
Why that?

Reply:
`unpin_dt` and `channel_id.last_interest_dt` wil have the same value and then the is_pinned will be messed up. actually, all the `dt` is here is the same, which is not possible in real life.

Reply:
nope, because the problem is at first, the channel should not be pinned for operator initially then when posting the message, the channel is pinned. if the dts are the same, we will break either of it even if we have the equal case.